release_notes/v0.8.md

```diff
@@ -0,0 +1,16 @@
+# What’s New in OPEA v0.8
+
+- Broaden functionality
+    - Supported frequently asked questions (FAQs) generation GenAI example
+    - Expanded the support of LLMs such as Llama3.1 and Qwen2 and supported LVMs such as llava
+    - Enabled end-to-end peformance and accuracy benchmarking
+    - Supported the experimental Agent microservice and GenAI example
+    - Supported LLM serving on Ray
+
+- Multi-platform support
+    - Released the docker images of GenAI components under OPEA dockerhub and supported the deployment with docker
```

Comment:
Also, can you add a link to the OPEA dockerhub here:
```suggestion
    - Releases the Docker images of GenAI components under OPEA dockerhub and supports the deployment with Docker
```

Reply:
It's title of PR, should we change it?